jest.setup.ts

```diff
@@ -1,6 +1,10 @@
+import { config } from 'dotenv'
 import 'jest-styled-components' // include style rules in snapshots
 import fetchMock from 'jest-fetch-mock' // Mocks `fetch` calls in unittests
 
+// For simplicity, we will use CowSwap default .env for all projects and libs
+config({ path: __dirname + '/apps/cowswap-frontend/.env' })
```

Comment:
Wow! I didn't know about this config

Reply:
In principle we can move the `.env` back to the root. I was tempted to do it, but didn't (i had too many changes already)

but i think we should